test/migrations/V9000__test_data.sql

```diff
@@ -5,16 +5,21 @@ INSERT INTO fileqc (fileid, fileswid, md5sum, project, workflow, filepath, qcpas
 ('vidarr:research/file/000020f1ddaa79c72ca761b6fbc919a192c41f88062863757495774bbf9a6235', 12020, 'efd75ff3495dd6ffa8f84cf3fd3d0476', 'MultiSwid', 'MultiQC', '/analysis/extra/added', TRUE, 'me', 'fixed it'),
 ('vidarr:research/file/dfa067d1dc34c3f4b9ab4da72f03a5ca541e7b4582f8f1e855a56aab1d2c2e4', 12016, 'b2d0e43098750b3a7cd97c7a8e0b5eec', 'DeleteMe', 'CASAVA', '/delete/me', FALSE, 'mistaken', NULL);
 
-DELETE FROM cardea_case;
-INSERT INTO cardea_case (case_identifier, requisition_id, lims_ids) VALUES
-('R11_TEST_1000_Xy_Z', 11, '{"109_1_LDI5432", "109_1_LDI4321"}'),
-('R12_TEST_1212_Ab_C', 12, '{"901_1_LDI9001", "901_1_LDI9002", "902_1_LDI9001", "902_1_LDI9002"}'),
-('R13_TEST_9999_De_F', 13, '{"109_1_LDI1234", "109_1_LDI4321"}');
+DELETE FROM archive_entity;
+INSERT INTO archive_entity (entity_identifier, requisition_id, lims_ids, entity_type) VALUES
+('R11_TEST_1000_Xy_Z', 11, '{"109_1_LDI5432", "109_1_LDI4321"}', 'CASE'),
+('R12_TEST_1212_Ab_C', 12, '{"901_1_LDI9001", "901_1_LDI9002", "902_1_LDI9001", "902_1_LDI9002"}', 'CASE'),
+('R13_TEST_9999_De_F', 13, '{"109_1_LDI1234", "109_1_LDI4321"}', 'CASE'),
+('P13_TEST_1000_De_F', 14, '{"109_1_LDI5432", "109_1_LDI4321"}', 'PROJECT'),
+('P14_TEST_1313_Wx_Y', 15, '{"901_1_LDI9001", "901_1_LDI9002", "902_1_LDI9001", "902_1_LDI9002"}', 'PROJECT'),
+('P15_TEST_8888_Lm_N', 16, '{"109_1_LDI1234", "109_1_LDI4321"}', 'PROJECT');
```

Comment:
Would you consider changing these project identifiers to look more like project codes in MISO?